Add Elastic Agent 8.19.19 release notes (#15680)
elastic/elastic-agent: changelog/8.19.19.asciidoc

// begin 8.19.19 relnotes

[[release-notes-8.19.19]]
==  8.19.19

Review important information about the  8.19.19 release.

[discrete]
[[security-updates-8.19.19]]
=== Security updates

> * Redact sensitive HTTP header values embedded in Fleet env vars in diagnostics archives. https://github.com/elastic/elastic-agent/pull/15284[#15284] 

Reviewer: ```suggestion
* Redact sensitive HTTP header values embedded in Fleet environment variables in diagnostics archives. https://github.com/elastic/elastic-agent/pull/15284[#15284] 
```

[discrete]
[[enhancements-8.19.19]]
=== Enhancements

* Update OTel Collector components to v0.155.0. https://github.com/elastic/elastic-agent/pull/15412[#15412] 

[discrete]
[[bug-fixes-8.19.19]]
=== Bug fixes

* Fix duplicate entries, empty unit dirs, and EDOT error handling in OTel diagnostics. https://github.com/elastic/elastic-agent/pull/15108[#15108] 

> * Preserve locally-configured monitoring.http.host across Fleet policy check-ins. https://github.com/elastic/elastic-agent/pull/15291[#15291] 

Reviewer: ```suggestion
* Preserve locally-configured `monitoring.http.host` across Fleet policy check-ins. https://github.com/elastic/elastic-agent/pull/15291[#15291] 
```

> * Fix spurious &#34;failed to unmarshal checkin actions&#34; error on idle Fleet check-ins. https://github.com/elastic/elastic-agent/pull/15398[#15398] https://github.com/elastic/elastic-agent/issues/15397[#15397]

Reviewer: ```suggestion
* Fix "failed to unmarshal checkin actions" error on idle Fleet check-ins. https://github.com/elastic/elastic-agent/pull/15398[#15398] https://github.com/elastic/elastic-agent/issues/15397[#15397]
```

> * Override fleet.ssl.certificate_authorities from env vars in container mode. https://github.com/elastic/elastic-agent/pull/15427[#15427] 

Reviewer: ```suggestion
* Override `fleet.ssl.certificate_authorities` from environment variables in container mode. https://github.com/elastic/elastic-agent/pull/15427[#15427] 
```
